### PR TITLE
Update the meta IDL

### PR DESCRIPTION
**Pull request by ShanniLi**
For #9

Per our discussion.
r @Raynos @kriskowal 
cc @prashantv @breerly @abhinav @blampe @junchaowu

### CODE REVIEW

**Comment by abhinav** [comment]
:+1:

**Comment by junchaowu** [comment]
lgtm

**Comment by Raynos** [comment]
lgtm.

File: meta.thrift [hunk 1]
```diff
@@ -1,4 +1,9 @@
+struct HealthStatus {
+    1: required bool ok
+    2: optional string message
```
Comment (breerly) [inline meta.thrift]: What's the use-case for this?

Comment (breerly) [inline meta.thrift]: I suppose saying what is not ok.

Comment (ShanniLi) [inline meta.thrift]: yes, as suggested by @prashantv

**Comment by kriskowal** [comment]
woot

**Pull request merged**